distributed/diagnostics/tests/test_nvml.py

```diff
@@ -17,11 +18,15 @@ def test_one_time():
 def test_1_visible_devices():
     os.environ["CUDA_VISIBLE_DEVICES"] = "0"
     output = nvml.one_time()
-    assert len(output["memory-total"]) == 1
+    h = nvml._pynvml_handles()
+    assert output["memory-total"] == pynvml.nvmlDeviceGetMemoryInfo(h).total
 
 
 @pytest.mark.parametrize("CVD", ["1,0", "0,1"])
 def test_2_visible_devices(CVD):
+    if pynvml.nvmlDeviceGetCount() <= 1:
+        pytest.skip("Machine only has a single GPU")
+
```

Comment:
Added this to skip the multi-GPU test on single GPU machines based on our conversation offline, can you please take a look and see if I got this right, Ben? :)